latest/index.bs

```diff
@@ -331,24 +317,917 @@ Conforming readers:
+│   └── displacements           # for example, a zarray containing a displacement field
+│       ├── .zattrs
+│       └── .zarray
+│
+├── volume
+│   ├── .zattrs                 # group level attributes (multiscales)
+│   └── 0                       # a group containing the 0th scale
+│       └── image               # a zarr array
+│           ├── .zattrs         # physical coordinate system and transformations here
+│           └── .zarray         # the array attributes
+└── crop
+    ├── .zattrs                 # group level attributes (multiscales)
+    └── 0                       # a group containing the 0th scale
+        └── image               # a zarr array
+            ├── .zattrs         # physical coordinate system and transformations here
+            └── .zarray         # the array attributes
+</pre>
+
+### Additional details
+
+Most coordinate transformations MUST specify their input and output coordinate systems using `input` and `output` with a string value
+corresponding to the name of a coordinate system. The coordinate system's name may be the path to an array, and therefore may
+not appear in the list of coordinate systems.
+
+Exceptions are if the the coordinate transformation appears in the `transformations` list of a `sequence` or is the
+`transformation` of an `inverseOf` transformation. In these two cases input and output SHOULD be omitted (see below for
+details).
+
+Transformations in the `transformations` list of a `byDimensions` transforemation MUST provide `input` and `output` as arrays
+of strings corresponding to axis names of the parent transformation's input and output coordinate systems (see below for
+details).
+
+<div class=example>
+
+The sequence transformation's input corresponds to an array coordinate system at path "/my/array".
+
+```json
+"coordinateSystems" : [
+    { "name" : "in", "axes" : [{"name" : "i"}, {"name":"j"}] },
+    { "name" : "outScale", "axes" : [{"name" : "x"}, {"name":"y"}] },
+    { "name" : "outSeq", "axes" : [{"name" : "x"}, {"name":"y"}] },
+    { "name" : "outInv", "axes" : [{"name" : "x"}, {"name":"y"}] },
+    { "name" : "outByDim", "axes" : [{"name" : "x"}, {"name":"y"}] }
+],
+"coordinateTransformations" : [
+    {
+        "type": "scale",
+        "input" : "in",
+        "output" : "outSeq",
+        "scale" : [ 0.5, 1.2 ]
+    },
+    {
+        "type" : "sequence",
+        "input" : "/my/array",
+        "output" : "outSeq",
+        "transformations" : [
+            { "type": "scale", "scale" : [ 0.5, 0.6 ] },
+            { "type": "translation", "translation" : [ 2, 5 ] }
+        ]
+    },
+    {
+        "type": "inverseOf",
+        "input" : "in",
+        "output" : "outInv",
+        "transformation" : {
+            "type": "displacements",
+            "path": "/path/to/displacements"
+        }
+    },
+    {
+        "type": "byDimension",
+        "input" : "in",
+        "output" : "outDim",
```

Comment:
Should this `outDim` match the `"name" : "outByDim"` from the coordinateSystems list above?